test/e2e/framework/test_context.go

```diff
@@ -349,24 +349,6 @@ func RegisterClusterFlags(flags *flag.FlagSet) {
 	flags.DurationVar(&nodeKiller.SimulatedDowntime, "node-killer-simulated-downtime", 10*time.Minute, "A delay between node death and recreation")
 }
 
-// RegisterNodeFlags registers flags specific to the node e2e test suite.
-func RegisterNodeFlags(flags *flag.FlagSet) {
-	// Mark the test as node e2e when node flags are api.Registry.
-	TestContext.NodeE2E = true
-	flags.StringVar(&TestContext.NodeName, "node-name", "", "Name of the node to run tests on.")
-	// TODO(random-liu): Move kubelet start logic out of the test.
-	// TODO(random-liu): Move log fetch logic out of the test.
-	// There are different ways to start kubelet (systemd, initd, docker, manually started etc.)
-	// and manage logs (journald, upstart etc.).
-	// For different situation we need to mount different things into the container, run different commands.
-	// It is hard and unnecessary to deal with the complexity inside the test suite.
-	flags.BoolVar(&TestContext.NodeConformance, "conformance", false, "If true, the test suite will not start kubelet, and fetch system log (kernel, docker, kubelet log etc.) to the report directory.")
-	flags.BoolVar(&TestContext.PrepullImages, "prepull-images", true, "If true, prepull images so image pull failures do not cause test failures.")
```

Comment:
This PrepullImages is used in e2e_node/system_node_critical_test.go, e2e_node/e2e_node_suite_test.go, e2e_node/eviction_test.go and e2e/framework/pods.go
We need to take more time for digging part of `e2e/framework/pods.go`

Reply:
This PrepullImages also is used if the test is e2e_node test like
e2e/framework/pods.go:
```
181 // mungeSpec apply test-suite specific transformations to the pod spec.
182 func (c *PodClient) mungeSpec(pod *v1.Pod) {
183         if !TestContext.NodeE2E {
184                 return
185         }
186 
187         gomega.Expect(pod.Spec.NodeName).To(gomega.Or(gomega.BeZero(), gomega.Equal(TestContext.NodeName)), "Test misconfigured")
188         pod.Spec.NodeName = TestContext.NodeName
189         // Node e2e does not support the default DNSClusterFirst policy. Set
190         // the policy to DNSDefault, which is configured per node.
191         pod.Spec.DNSPolicy = v1.DNSDefault
192 
193         // PrepullImages only works for node e2e now. For cluster e2e, image prepull is not enforced,
194         // we should not munge ImagePullPolicy for cluster e2e pods.
195         if !TestContext.PrepullImages {
196                 return
197         }
```
So this flags also can be moved into e2e_node test.